Source/ARTPresenceMessage.h

```diff
@@ -8,7 +8,7 @@
 #import <Ably/ARTBaseMessage.h>
 #import <Ably/ARTEventEmitter.h>
 
-/// Presence action type
+/// ARTPresenceAction is an enum representing all the Realtime Presence states & events.
```

Comment:
So action ~== states ~== events? _Why_ are using 3 words for the same concept? 😅

Maybe this would be clearer:
> ARTPresenceAction represents all actions an ARTPresenceMessage can indicate.

Reply:
https://github.com/ably/ably-cocoa/pull/1176/commits/ff80ee88ab513d50518e33f823b70a41213d1b7f